status/projects.py

```diff
@@ -627,3 +627,14 @@ def get(self, projectname):
                         print "cannot add {} to paths, one of these two keys does not exist: sample->{} run->{}".format(os.path.relpath(os.path.join(root,f), prefix), cursample, currun)
         self.set_header("Content-type", "application/json")
         self.write(json.dumps(paths))
+
+class CharonProjectHandler(SafeHandler):
+    """queries charon about the current project"""
+    def get(self, projectid):
+        url="http://charon.scilifelab.se/api/summary?projectid={}".format(projectid)
```

Comment:
what about a section:

```yaml
charon:
    url: http://charon.scilifelab.se/api/
```

In the config file so you don't have to remember where in the code is the URL in the case it changes?